internal/table/timeseries/timeseries.go

```diff
@@ -283,7 +285,7 @@ func (t *Table) Append(block block.Block) error {
 	t.schema.Store(block.Schema())
 
 	// Append the block to the store
-	return t.store.Append(key.New(string(block.Key), time.Unix(0, ts)), buffer, t.ttl)
```

Comment:
@tardunge Is it a bug before?

Reply:
No. I'm just giving time.Now() there.

Reply:
I'm sorry. Yes it is a bug from before. This is a temp fix.